### DISCUSSION TITLE
有没有大佬做聚合算法的

### DISCUSSION THREAD

**Discussion by DracoYan-111** [rebase-network/hello-world#22] [Ideas]
想做一个聚合算法，该合约将以插件的形式在其他的网站上进行快捷的swap操作，将以收取手续费的形式获利

**Comment by luckyyang** [comment]
不错，可以在哪些网站操作呢

**Reply by DracoYan-111** [reply]
好问题，因为现在还在研发阶段，这个需要人家引入我们放在ipfs上的代码，我们相信我们运营的力量

**Reply by luckyyang** [reply]
类似于将 1inch 的功能嵌入到所有的网站中吗

**Reply by DracoYan-111** [reply]
对，但不仅仅是1inch，还有聚合其他swap平台
